README.md

```diff
@@ -8,8 +8,8 @@
 
 ## Description
 
-The **Otter** Project has the goal to provide a common platform to accelerate and facilitate the development on web application based on Angular.
-Otter is a very module framework split in several unit working together on a common base allowing CMS customization.
+The **Otter** Project has the goal to provide a common platform to accelerate and facilitate the development on web applications based on Angular.
+Otter is a highly modular framework split in several units working together on a common base allowing CMS customization.
```

Comment:
This sentence seems a bit long. Do we really care for that much details? What is the message here ?

Proposal to be validated with the rest of the team:
Otter is a highly modular framework. It is split into several units to cover different aspects of an angular web application (localization, testing, customization etc.). It also supports the integration to a CMS thanks to generated metadata from the application source code analysis.